jaggeryapps/admin/serviceproviders/addsp.jag

```diff
@@ -198,90 +222,12 @@ include("../templates/footer.jag");
     $(document).on('click', '.cloud-app-type', function(){
```

Comment:
Shall we move this script block to a js file rather keeping in the jag file itself?